packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md

```diff
@@ -423,7 +444,12 @@ If a user is suspended, deactivated, or removed from the application in Okta, th
 
 **POST** /Groups
 
-To create a group on the SCIM server, an Okta administrator must push the group using the Okta Admin Console. In the Okta Admin Console, select the SCIM application from your list of applications. On the **Push Groups** tab, click **Push Groups**. You can select which existing Okta group to push, either by specifying a name or a rule. For more information, see the [Using Group Push topic](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Directory_Using_Group_Push) in the Okta Help Center.
+To create a group on the SCIM server, an Okta administrator must push the group using the Okta Admin Console. In the Okta Admin Console:
+
+* Select the SCIM application from your list of applications
```

Comment:
These are steps that need to be numbered and need periods at the end of each.

Reply:
Fixed.